appinventor/blocklyeditor/src/exportBlocksImage.js

```diff
@@ -461,13 +462,23 @@ Blockly.exportBlockAsPng = function(block) {
     img.src = uri;
     img.onload = function() {
       var canvas = document.createElement('canvas');
-      canvas.width = img.width;
-      canvas.height = img.height;
+      canvas.width = 2 * img.width;
+      canvas.height = 2 * img.height;
       var context = canvas.getContext('2d');
-      context.drawImage(img, 0, 0);
+      context.drawImage(img, 0, 0, img.width, img.height, 0, 0, canvas.width, canvas.height);
 
       function download(png) {
         png.setCodeChunk(code);
+        for (var i = 0; i < png.chunks.length; i++) {
+          var phy = [112, 72, 89, 115];
+          if (png.chunks[i].type == 'pHYs') {
+            png.chunks.splice(i, 1, new PNG.Chunk(9, 'pHYs', pHY_data, crc32(phy.concat(pHY_data)))); //replacing existing pHYs chunk
+        	break;
```

Comment:
Not sure why the break is indented here. It should be at the same indentation as the previous line.